recipes/pipenv/LICENSE

```diff
@@ -0,0 +1,22 @@
+The MIT License (MIT)
```

Comment:
This file isn't needed, `conda-build` will find it in the source tarball.

(The `licence_file` entry is still needed)